package.json

```diff
@@ -763,6 +768,13 @@
                     "when": "view == azureApiManagementExplorer && viewItem == azureApiManagementAuthorizationAccessPolicy",
                     "group": "3@1"
                 }
+            ],
+            "copilot": [
+                {
+                    "command": "azureApiManagement.explainPolicy",
+                    "when": "github.copilot-chat.activated && resourceScheme == 'file'",
```

Comment:
Only show the command when file is xml? 
I am not sure whether it is possible to check '*.policy.xml'. If it is possible, it will be even better. :)

Reply:
Thank you. Since I expect users can use this functionality with their own policy files, and not sure what file extension (cshtml, xml, others?) they will use. Do you think it makes sense to show in each file first? I created a task to follow up with PM on this. May add telemetry to collect more info if it's hard for us to make decisions.